### ISSUE TITLE
Stack trace from Plugin Exception

### ISSUE THREAD

**Issue by RileyMShea** [tonybaloney/pycharm-security#33]
This occurred when I had a React project open and no python interpreter active.

> java.lang.NullPointerException
	at security.packaging.PyPackageSecurityScan.renderMessage(PyPackageSecurityScan.kt:80)
	at security.packaging.PyPackageSecurityScan.checkPackages(PyPackageSecurityScan.kt:55)
	at security.packaging.PythonPackageVulnerabilityStartupTask$runActivity$1.run(PythonPackageVulnerabilityStartupTask.kt:19)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

**Comment by tonybaloney** [comment]
hi @RileyMShea 

thanks for reporting this issue.

Which version of the plugin are you using? 
Which editor are you running and which version (e.g. PyCharm 2019.3.2)

**Comment by tonybaloney** [comment]
From your other report, it looked like you're running 1.2.0.

The issue was:
I have reproduced this where it found a package with a security issue, but no CVE entry. The model for safetydbrecord had incorrectly set cve to a string type (not-nullable)
Kotlin has nullable strings as a separate type. Java just has strings that can take null values.
Because this uses a Java package for deserializing the database. The kotlin compiler was happy but the execution still created an NPE.
Fixed in PR
